docs/start/install.rst

```diff
@@ -247,7 +385,22 @@ know how to resolve them. If none of these help you, please ask a question on
   The most likely cause of this error is a misconfigured ``PYTHONPATH``
   environment variable. Unset that variable and try again.
 
+``ModuleNotFoundError: No module named 'gendoc'``
+  Check your python version with ``python --version``. if you are running 
+  ``python3.10``, you will need to :ref:`install the latest dev version<Install Dev Version>`
+  or :ref:`build from source<Install from Source>` or revert to ``python3.9``.
+
+  - This issue is resolved by `Include gendoc file into the source distribution (#3151)`_ is
+    currently available in the :ref:`latest development version<Install Dev Version>`.
```

Comment:
There are quite a lot of bugs that have been fixed from the time we released datatable `1.0.0`, I'm not sure if we want to list them all here...